cardano-wallet.cabal

```diff
@@ -66,6 +70,22 @@ library
       Cardano.Wallet
       Cardano.Wallet.AddressDerivation
       Cardano.Wallet.AddressDiscovery
+      Cardano.Wallet.Api.V2
+      Cardano.Wallet.Api.V2.JSON
+      Cardano.Wallet.Api.V2.Lenses
+      Cardano.Wallet.Api.V2.Types.Amount
+      Cardano.Wallet.Api.V2.Types.CurrencyUnit
+      Cardano.Wallet.Api.V2.Types.Percentage
+      Cardano.Wallet.Api.V2.Types.Wallet
+      Cardano.Wallet.Api.V2.Types.WalletAddressPoolGap
+      Cardano.Wallet.Api.V2.Types.WalletBalance
+      Cardano.Wallet.Api.V2.Types.WalletDelegation
+      Cardano.Wallet.Api.V2.Types.WalletDelegationStatus
+      Cardano.Wallet.Api.V2.Types.WalletId
+      Cardano.Wallet.Api.V2.Types.WalletName
+      Cardano.Wallet.Api.V2.Types.WalletPassphraseInfo
+      Cardano.Wallet.Api.V2.Types.WalletState
+      Cardano.Wallet.Api.V2.Types.WalletStateStatus
```

Comment:
Hehe. Many, Tiny, Modules. May we group that into one module called `Types`, unless there's a good reason to split something in a separate one. Again, we've done that, aaand, having many modules that live within the same "realm" and are at the same layers doesn't work well. Modules get rapidly inconsistent and, it's quite painful to maintain and have to jump between all of them when making a change. Having one module with all the API types is fine.